Back-End/controllers/registration_controllers/register_controllers.js

```diff
@@ -1,5 +1,48 @@
-const db = require('../../db');
+const connection = require('../../db');
+const bcrypt = require('bcrypt');
+
+const  register =(req,res)=>{
+  const{name,adress,email,password,phone} =req.body
+  let role_id =0
+
+    //setting the role for the registered account
+    if(req.params.role ==="instructor"){
+        role_id = 2
+     }else if(req.params.role ==="student"){
+        role_id = 3
+     };
```

Comment:
Explain how are you handling the roles

Reply:
that's on checkbox at react

Reply:
You need to make it dynamic because what if you modify the schema file and another role gets added before the instructor role, that will result in the instructor role to be id 3 since the id is auto-increment, so it is better if you would get the id from the DB depending on the role